index.js

```diff
@@ -7,6 +7,7 @@ export default class ClapprStats extends ContainerPlugin {
   get name() { return 'clappr_stats' }
 
   get _playbackName() {return this.container.playback.name}
+  get _currentTime() {return this.container.playback.getCurrentTime()}
```

Comment:
My problem with this is that is only applicable to hls and html5, @clappr/core maybe this should be added to the Playback API, thoughts?